app/helpers/db_flavor.py

```diff
@@ -17,3 +18,27 @@
         'port': 3306
     }
 }
+
+# Database flavours
+database_flavours = [
```

Comment:
Using a dictionary here instead of a list would simplifier look ups for this operation.